sprout/appliances/models.py

```diff
@@ -2,12 +2,7 @@
 import base64
 import re
 import yaml
-import six
-
-try:
-    import six.moves.cPickle as pickle
-except ImportError:
-    import pickle   # NOQA
+import pickle   # NOQA
```

Comment:
can we remove NOQA?

Reply:
:D

Reply:
yeah I'm wondering exactly why its there

Reply:
probably because of overeager shadowing detection ^^